specification/communication/Communication.Messages/models.tsp

```diff
@@ -0,0 +1,370 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+using Azure.ClientGenerator.Core;
+
+@versioned(AzureCommunicationMessagesService.Versions)
+namespace AzureCommunicationMessagesService;
+
+@access(Access.public)
```

Comment:
This all should default to public, so all these declarations are extraneous, I believe.

Reply:
removed